task5/src/main/java/com/clouway/adapter/http/LoginController.java

```diff
@@ -0,0 +1,59 @@
+package com.clouway.adapter.http;
+
+import com.clouway.core.RandomGenerator;
+import com.clouway.core.Session;
+import com.clouway.core.SessionRepository;
+import com.clouway.core.UserRepository;
+import com.clouway.core.Validator;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.Cookie;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+/**
+ * Created by Kristiyan Petkov  <kristiqn.l.petkov@gmail.com> on 27.05.16.
+ */
+@WebServlet(name = "LoginController")
+public class LoginController extends HttpServlet {
```

Comment:
Please do not delete and create same files. I loose comment history this way. Use cut and paste. Do not know how you did it for example in this file there should be couple comments.